pkg/operator/ceph/csi/spec.go

```diff
@@ -370,6 +377,17 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	pluginNodeAffinity := getNodeAffinity(clientset, false)
 	if rbdPlugin != nil {
 		applyToPodSpec(&rbdPlugin.Spec.Template.Spec, pluginNodeAffinity, pluginTolerations)
+		// get resource details for rbdplugin
+		rbdResource := getComputeResource(clientset, rbdPluginResource)
```

Comment:
Seems like lines 381-390 could be factored out to avoid duplication. How about a helper method such as `applyResourcesToContainers()`?